www/viewgame

```diff
@@ -1734,7 +1937,87 @@ if ($curuser) {
         list($embargoID, $embargoDate) = mysql_fetch_row($result);
 }
 
+// --------------------PHP Functions for Displaying Median Play Time--------------------------
+
+
+// Find the median of the numbers in an array
+function findMedian($array_input) {
+   if(is_array($array_input)) {
+       // Sort the entries in the array
+       sort($array_input);
+       // Divide the number of entries by two
+       $total_entries = count($array_input);
+       $half = $total_entries / 2;
+       $int_half = (int) $half;      // Find just the integer part of a number, in case it's not a whole number
+
+       // Calculate median 
+       if ($total_entries % 2 != 0) {            // Dividing by 2 gives a remainder, so the array must have an odd number of entries
+        $median = $array_input[$int_half];	     // Choose the middle entry
+
+       } else {                                  // The array has an even number of entries, so there will be two middle entries
+       $middle1 = $array_input[$int_half-1];     // Find the first middle entry
+       $middle2 = $array_input[$int_half];      // Find the second middle entry
+       $median = ($middle1 + $middle2) / 2;
+       }
+   return $median;
+   }
+}
+
+// Get all of the estimated play times for this game, and put them in an array
+$result = mysqli_execute_query($db, "select time_in_minutes from playertimes where gameid = ?", [$id]);
+if (!$result) throw new Exception("Error: " . mysqli_error($db));
+$alltimes= array();
+while ($row = mysqli_fetch_assoc($result)) {
+    $time=$row['time_in_minutes'];
+    $alltimes[]=$time;
+}
+
+// Find the median play time for this game, and separate the hous and minutes
+$mediantime=findMedian($alltimes);
+$median_hours_and_minutes=convertToHoursAndMinutes($mediantime);
+$median_hours=$median_hours_and_minutes['hours'];
+$median_minutes=$median_hours_and_minutes['minutes'];
+
+// For the official estimated time for a game, we we want to round to the 
+// nearest 5 minutes when the game is longer than 1 hour.
+$official_estimated_time_text="";
+if ($median_minutes != 0) {
+    if ($median_hours >= 1 ) {
+        // If the game is over an hour, round the minutes to the nearest 5 minutes. 
+	$minutes_rounded_to_five = round( $median_minutes / 5 ) * 5;
+        $official_estimated_time_text = convertTimeToText(['hours' => $median_hours, 'minutes' => $minutes_rounded_to_five]);
+    } else {
+        // If the game is under an hour, round the minutes to the nearest whole number.
+        $whole_minutes = round($median_minutes);
+        $official_estimated_time_text = convertTimeToText(['hours' => $median_hours, 'minutes' => $whole_minutes]);
+    }
+} else {
+    $official_estimated_time_text=convertTimeToText($median_hours_and_minutes);
+}
+
+
+
+function displayMedianTime($median_hours, $median_minutes, $official_estimated_time_text, $alltimes) {
+    if (($median_hours >= 1) or ($median_minutes >= 1)) {
```

Comment:
return immediately when this is false, so the rest of the code can be unindented